devices/enocean.js

```diff
@@ -28,7 +28,8 @@ module.exports = [
         exposes: [e.action(['press_1', 'release_1', 'press_2', 'release_2', 'press_3', 'release_3', 'press_4', 'release_4',
             'press_1_and_2', 'release_1_and_2', 'press_1_and_3', 'release_1_and_3', 'press_1_and_4', 'release_1_and_4',
             'press_2_and_3', 'release_2_and_3', 'press_2_and_4', 'release_2_and_4', 'press_3_and_4', 'release_3_and_4',
-            'press_energy_bar', 'release_energy_bar', 'press_or_release_all'])],
+            'press_energy_bar', 'release_energy_bar', 'press_or_release_all',
+            'lock', 'unlock', 'half_open', 'tilt'])],
```

Comment:
The action is not in here: https://github.com/Koenkk/zigbee-herdsman-converters/blob/06b3a4c68fed36c822e419a63049b7319e663ecd/converters/fromZigbee.js#L4820 so it these actions will never be produced.

Reply:
Hey Koenkk,
I work in a factory where we product some connected handle, we work with Dolphin (Enocean) so we use their PTM but not PTM215 or 216 but PTM535. they are the same
https://perpetuum.enocean.com/01-2020-en/la-croisee-ds-connected-handle-for-sliding-doors-and-windows/?lang=en
So now we work with the PTM535BZ, similar of PTM216 or 215 and we program it to send 0x50, 0x51, 0x52, and 0x53 for the command send code.
so as the zigbe alliance said in their document 0x50 is for lock, 0x51 is for unlock, 0x52 is for half-open ( in french oscillo-battant) and 0x54 is a tilt sensor for detect the shock in the windows.
we are not ready for the moment to sell our new zigbee product but we need thes code to test it.
Best regards
Long life to Z2M
![image](https://user-images.githubusercontent.com/83968695/160081393-77f0c13c-0bd9-41a0-934e-be344b5a2b7f.png)